purefb/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - PureFB
 
+## 2.0.1 / 2024-07-16
```

Comment:
Technically this is a breaking change, consider bumping the major version.
I.e. anyone relying on the old name, however wrong it may be, will have their dashboards/monitors broken by this release.

Not blocking.